### ISSUE TITLE
breadcrumb link - insufficient contrast

### ISSUE THREAD

**Issue by tlholt02** [samvera/hyku#2878]
Hyku v7.0.0 with Hyrax v5.2.0 Homepage https://demo.hykudemo.org/ 

The link(s) in the breadcrumb have insufficient contrast. Contrast ratio is 4.16. For a font-size of 12pt it needs to be at least 4.5. Recommendation: make background-color white and use border-bottom to separate it visually from page content.

([WCAG 2.1 AA 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html))
